plugins/k8s/resoto_plugin_k8s/deferred_edges/azure.py

```diff
@@ -0,0 +1,60 @@
+from resotolib.baseresources import BaseResource
+from resotolib.graph import BySearchCriteria, ByNodeId, Graph
+from resoto_plugin_k8s.deferred_edges.utils import rgetattr
+
+
+def link_k8s_node_to_vmss_instance(graph: Graph, resource: BaseResource) -> None:
+    if resource.kind == "kubernetes_node":
+        if (labels := rgetattr(resource, "labels", {})) and (nodegroup := labels.get("kubernetes.azure.com/cluster")):
+            graph.add_deferred_edge(
+                BySearchCriteria(f"is(azure_managed_cluster) and reported.node_resource_group={nodegroup}"),
+                ByNodeId(resource.chksum),
+            )
```

Comment:
The node is attachted to the kubernetes cluster.
The kubernetes cluster is attached to the AKS cluster.
There is no need to create an edge from the node to the AKS cluster.